### ISSUE TITLE
Line Numbers?

### ISSUE THREAD

**Issue by robertjpayne** [dechaoqiu/atosl#1]
I'm trying to get atosl to work much like Apple's provided atos but not having a ton of luck. I get the OBJC class/method name out of the program but can't figure out how to get line numbers. Is this expected? Would love to help in any way I can!

**Comment by dechaoqiu** [comment]
You can get line number information for your own Objective-C code. If the class/method is from the objective-C library, then the line number information has already been strip from the original binary macho file.

**Comment by robertjpayne** [comment]
True, but the DWARF files contain this information, it's something Apple atos can do which is extremely helpful for debugging potentially long methods.

I just wasn't sure if it was supposed to be printing them off or not.

**Comment by dechaoqiu** [comment]
Can you give me a link to your sample Mach-O file and the atosl output?
There is a folder containing test file in this project, can you get the correct result in your machine?
![image](https://cloud.githubusercontent.com/assets/1616053/2858010/7266b5a4-d17f-11e3-80f1-e43053066f62.png)
